### PR TITLE
Fix typos (user-facing and non-user-facing

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -L datas,medias,warmup`

### CODE REVIEW

File: tests/utils/__init__.py [hunk 1]
```diff
@@ -83,7 +83,7 @@ class Encoder:
     def __init__(self):
         self.substitutions = {}
 
-    def subsitute(self, data: bytes) -> str:
+    def substitute(self, data: bytes) -> str:
```
Comment (luzpaz) [inline tests/utils/__init__.py:86]: please review closely

File: tests/utils/__init__.py [hunk 2]
```diff
@@ -110,7 +110,7 @@ def write(self, data):
         try:
             self.original_buffer.write(data.decode(UTF8))
         except UnicodeDecodeError:
-            self.original_buffer.write(self.encoder.subsitute(data))
+            self.original_buffer.write(self.encoder.substitute(data))
```
Comment (luzpaz) [inline tests/utils/__init__.py:113]: ditto

**Review by isidentical — APPROVED** [review]
LGTM

**Review by petersadek90 — COMMENTED** [review]
Thnx

**Review by petersadek90 — COMMENTED** [review]
Thnx

**Review by petersadek90 — COMMENTED** [review]
Thnx

**Pull request merged**